packages/dev/core/src/Shaders/ShadersInclude/pbrDebug.fx

```diff
@@ -128,6 +128,14 @@ if (vClipSpacePosition.x / vClipSpacePosition.w >= vDebugMode.x) {
         gl_FragColor.rgb = subSurfaceOut.transmittance;
     #elif DEBUGMODE == 70 && defined(SUBSURFACE) && defined(SS_REFRACTION)
         gl_FragColor.rgb = subSurfaceOut.refractionTransmittance;
+    #elif DEBUGMODE == 72
+        gl_FragColor.rgb = vec3(1.0-roughness); // TODO
+    #elif DEBUGMODE == 73
+        gl_FragColor.rgb = vAlbedoColor.rgb; // TODO
+    #elif DEBUGMODE == 74
+        gl_FragColor.rgb = vReflectivityColor.rgb; // TODO
+    #elif DEBUGMODE == 75
+        gl_FragColor.rgb = vEmissiveColor.rgb; //TODO

```

Comment:
Similar to albedo, this is the color from the material only. Is it the one you intend to display :-)

The full premultiplication and gamma conversions can be found there https://github.com/BabylonJS/Babylon.js/blob/841bdc70820fe8c26f369e60964c7dac8543669d/packages/dev/core/src/Shaders/ShadersInclude/pbrBlockFinalUnlitComponents.fx

Also this would benefit from the #define DEBUGMODE_GAMMA cause the emissive color is in linear space.